### ISSUE TITLE
4.0 fails to run on python 2.7.10

### ISSUE THREAD

**Issue by sijis** [errbotio/errbot#667]
This is a fresh installation that went flawless. Executing `errbot` results in this error:
```
[saviles:/tmp/errbot] [tmp-4f679abd10dbe26a] 1 $ errbot 
Traceback (most recent call last):
  File "/home/saviles/virtualenvs/tmp-4f679abd10dbe26a/bin/errbot", line 9, in <module>
    load_entry_point('errbot==4.0.0', 'console_scripts', 'errbot')()
  File "/home/saviles/virtualenvs/tmp-4f679abd10dbe26a/lib/python2.7/site-packages/pkg_resources/__init__.py", line 519, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/home/saviles/virtualenvs/tmp-4f679abd10dbe26a/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2630, in load_entry_point
    return ep.load()
  File "/home/saviles/virtualenvs/tmp-4f679abd10dbe26a/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2310, in load
    return self.resolve()
  File "/home/saviles/virtualenvs/tmp-4f679abd10dbe26a/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2316, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/home/saviles/virtualenvs/tmp-4f679abd10dbe26a/lib/python2.7/site-packages/errbot/err.py", line 25, in <module>
    from .plugin_wizard import new_plugin_wizard
  File "/home/saviles/virtualenvs/tmp-4f679abd10dbe26a/lib/python2.7/site-packages/errbot/plugin_wizard.py", line 100
    u"If you continue, these will be overwritten.".format(
    ^
IndentationError: unexpected indent
```

**Comment by trunet** [comment]
Fixed doing this on line 100 of plugin_wizard.py:
        print u"Warning: A plugin with this name was already found at {path}\n".format(
                path=os.path.join(directory, module_name+u".{py,plug}")
            )
        print u"If you continue, these will be overwritten."

**Comment by sijis** [comment]
It looks like this commit is related to my bug report:
https://github.com/errbotio/errbot/commit/62e5fea4ad1d2439fda2404bf77a64053d29b560

**Comment by gbin** [comment]
fixed on master and 4.0 branch as of 261fb65ce264f05e145ef205a2b1893233817b30
